cli/command/stack/kubernetes/remove.go

```diff
@@ -18,6 +19,9 @@ func RunRemove(dockerCli *KubeCli, opts options.Remove) error {
 		err := stacks.Delete(stack, &metav1.DeleteOptions{})
 		if err != nil {
 			fmt.Fprintf(dockerCli.Out(), "Failed to remove stack %s: %s\n", stack, err)
```

Comment:
Should we still print this error in this case? or should it be

```go
if err != nil && !err.IsNotFound(err) {
```

Reply:
hum 🤔